tokio-util/src/time/delay_queue.rs

```diff
@@ -818,38 +1086,41 @@ impl<T> futures_core::Stream for DelayQueue<T> {
 impl<T> wheel::Stack for Stack<T> {
     type Owned = usize;
     type Borrowed = usize;
-    type Store = Slab<Data<T>>;
+    type Store = SlabStorage<T>;
```

Comment:
I'm not too familiar with the `wheel::Stack` trait, so it may not be possible, but shouldn't these be `Key` as well rather than `usize`?